app/src/main/java/com/framgia/wsm/data/source/RequestRepository.java

```diff
@@ -92,7 +92,7 @@ public RequestRepository(RequestRemoteDataSource remoteDataSource) {
         for (int i = 0; i < 5; i++) {
             Request request = new Request();
             request.setCreatedAt("16/06/2017");
-            request.setStatus(1);
+            request.setStatus("pending");
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "pending" appears 5 times in this file; the first occurrence is on line 95
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals